Automattic/woocommerce-payments
# Update pay-for-order js config billing email

#### Changes proposed in this Pull Request
This PR updates the pay-for-order js config billing email to the session email

#### Testing instructions
1. Go to WCPay Dev -> Enable the Pay-for-order flow for WooPay
2. Create a pay-for-order order
3. Go to the pay-for-order page
4. Check in the console if your `billing_email` in the `window.wcpayConfig` is the correct one.

*

-------------------


**Post merge**
